### PR TITLE
PDF preview controls

### LINKED ISSUE #2954: Controls for PDF preview
The continuation of #2949 

Gmails controls are simple and cover all the need users have when previewing PDF files:

image

Implement:

- [ ] Page X / Y
- [ ] Zoom out
- [ ] Reset zoom (to 100%) - active when zoom is different from 100%
- [ ] Fit to width - active when zoom is 100%
- [ ] Zoom in

### PR DESCRIPTION
**Pull request by limonte**
Closes #2954

### CODE REVIEW

File: extension/chrome/elements/attachment_preview_pdf.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
```
Comment (tomholub) [inline extension/chrome/elements/attachment_preview_pdf.ts:1]: Sorry that we don't have an obvious place to put these. I'm thinking this file could go in https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/ui/ - it's not a view file, and it's UI related, so I think the UI folder will be a good home for it.

Comment (limonte) [reply extension/chrome/elements/attachment_preview_pdf.ts]: Done :+1:

**Review by tomholub — APPROVED** [review]
Thanks!

**Comment by tomholub** [comment]
It's excellent. Thanks again.

**Pull request merged**